### PR TITLE
CRUD implementation -- incomplete

### PR DESCRIPTION
**Pull request by Evan-CTL**

### CODE REVIEW

File: econplayground/main/views.py [hunk 1]
```diff
@@ -595,65 +597,108 @@ def get_success_url(self):
...
 
-        return context
+        ctx['bank_list'] = bank_list
+        ctx['all_count'] = bank_list.count()
+        ctx['bank_list'] = bank_list
+        ctx['all_count'] = bank_list.count()
+
+        return ctx
+
+        return ctx
```
Comment (nikolas) [inline econplayground/main/views.py:667]: Duplicate return statement here - this can be removed.

Comment (Evan-CTL) [reply econplayground/main/views.py:667]: Sorry. The rebase got a little messy, thought I caught all of it.

File: econplayground/main/views.py [hunk 2]
```diff
@@ -663,45 +701,402 @@ def form_valid(self, form):
...
     model = Question
     template_name = 'main/question_list.html'
+    is_assignment = False
+    is_question_bank = False
+    is_question = True
 
     def get_queryset(self):
-        return Question.objects.all()
+        return Question.objects.all().order_by('created_at')
+        return Question.objects.all().order_by('created_at')
```
Comment (nikolas) [inline econplayground/main/views.py:1055]: Duplicate return statement here can be removed.

File: econplayground/main/views.py [hunk 3]
```diff
@@ -663,45 +701,402 @@ def form_valid(self, form):
...
+    is_question_bank = False
+    is_question = True
+
+    def test_func(self):
+        return user_is_instructor(self.request.user)
 
     def get_success_url(self):
-        return reverse('question_bank', kwargs={'pk': self.object.pk})
+        return reverse('question_list')
+        return reverse('question_list')
```
Comment (nikolas) [inline econplayground/main/views.py:1099]: Duplicate return statement

File: econplayground/main/views.py (hunk 1 above)
Comment (nikolas) [reply econplayground/main/views.py]: No worries, that's what our review process is for after all!

**Comment by nikolas** [comment]
I have some minor adjustments to make, and I'm getting some errors on some of the new views, but that can all be handled with future changes. I think in general the structure is going in the right direction here, so I'll merge this to master.

**Pull request merged**